File: 프로그래머스/0/120841. 점의 위치 구하기/점의 위치 구하기.swift
import Foundation

func solution(_ dot:[Int]) -> Int {
    let x = dot[0]
    let y = dot[1]
    
    if 0 < x && 0 < y {
        return 1
    } else if x < 0 && 0 < y {
        return 2
    } else if x < 0 && y < 0 {
        return 3
    } else {
        return 4
    }
}

File: 프로그래머스/0/120841. 점의 위치 구하기/점의 위치 구하기.js
function solution(dot) {
    const [x, y] = dot;
    
    if (x > 0 && y > 0) {
        return 1; // 제1사분면
    } else if (x < 0 && y > 0) {
        return 2; // 제2사분면
    } else if (x < 0 && y < 0) {
        return 3; // 제3사분면
    } else if (x > 0 && y < 0) {
        return 4; // 제4사분면
    }
}
